ubuntu/16.04/usr/local/share/bootstrap/common_functions.sh

```diff
@@ -69,11 +69,6 @@ is_chown_forbidden() {
   return $?
 }
 
-is_vboxsf_mountpoint() {
-  grep -q "/app vboxsf" /proc/mounts
-  return $?
-}
-
```

Comment:
The separate treatment is to work around not being able to chown or vboxsf or NFS, or even chmod on vboxsf!